bin/rad-project

```diff
@@ -98,6 +98,16 @@
            _   (do (put-str! "You must select either 1 or 2.")
                    (exit! 1)))))
 
+(def setup-upstream!
+  "Creates a commit to set the upstream to master, if master does not exist yet"
+  (fn []
+    (match (process! "git" ["log" "master"] "")
```

Comment:
I would prefer `git show-ref refs/heads/master`. It does not print to stderr and is more robust